scripts/tag.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+# Copyright © 2025 Meroxa, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Get the directory where the script is located
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+# Source the check_semver.sh script using the script directory
+source "${SCRIPT_DIR}/check_semver.sh"
+
+TAG=$1
```

Comment:
I see we already have the function `get_spec_version`, maybe we could take the tag from `connector.yaml` instead of asking for a parameter? Otherwise the tag would get rejected anyway by the CI action.

Reply:
Makes sense!